dev-logs/2023-feb/upgrades/axum_core_upgrade_to_0_2_9

```diff
@@ -0,0 +1,6 @@
+axum-core upgraded to 0.2.9 because current version(0.2.4) considered as vulnareable
```

Comment:
There is a typo in `vulnareable`, should be `vulnerable`. Same typo is found in other dev-logs files